caktus_primer/static/stylus/form.styl

```diff
@@ -83,10 +87,19 @@ fieldset
         pointer-events: none
         transition: all 0.2s
 
-    input.populated:not([type=checkbox]):not([type=radio]):not([type=file]) + label, input:focus:not([type=checkbox]):not([type=radio]):not([type=file]) + label, input[readonly] + label,input[disabled] + label, input[readonly] + label
+    input.populated:not([type=checkbox]):not([type=radio]):not([type=file]) + label,
+    input:focus:not([type=checkbox]):not([type=radio]):not([type=file]) + label,
+    input[readonly] + label,
+    input[disabled] + label
```

Comment:
is this the same and just reformatted?

Reply:
If you look at the end of the deleted line above, there is an extra `input[readonly] + label`.  The reformatting is to hopefully be able to catch things like that.